NagorniyAleksandr/SQLTask/src/main/java/university/dao/converter/ToObjectConverter.java

```diff
@@ -0,0 +1,129 @@
+package university.dao.converter;
+
+import university.models.*;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ToObjectConverter {
+
+    public static Student getOneStudentFromResultSet(ResultSet resultSet) throws SQLException {
+        if (resultSet.next()) {
+            int studentId = resultSet.getInt(1);
```

Comment:
Please remember, that your logic depends on right order of table's colums

if someone will change the view of result table you will get a runtime error